modules/desktops/river.nix

```diff
@@ -63,7 +63,6 @@ with host;
 
             riverctl spawn systemctl --user import-environment                          # Fix GTK Load
 
-            ${monitor}                                                                  # Monitor
```

Comment:
Why is this being removed? It's normal this errors out if the hostName is not correct.